internal/backend/http_transport.go

```diff
@@ -23,8 +33,23 @@ func Transport() http.RoundTripper {
 		IdleConnTimeout:       90 * time.Second,
 		TLSHandshakeTimeout:   10 * time.Second,
 		ExpectContinueTimeout: 1 * time.Second,
+		TLSClientConfig: &tls.Config{
```

Comment:
Same here: Only set this if a filename is present.

Reply:
Done.